docs/FileInput.md

```diff
@@ -72,7 +72,7 @@ Files are accepted or rejected based on the `accept`, `multiple`, `minSize` and
 Equivalent of [the `accept` attribute of an `<input type="file">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept). `accept` must be a valid [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml), according to [input element specification](https://www.w3.org/wiki/HTML/Elements/input/file) or a valid file extension.
 
 ```jsx
-<FileInput source="attachments" accept="application/pdf">
+<FileInput source="attachments" accept={{ 'application/pdf': ['.pdf'] }}>
```

Comment:
Can you also add a mention about this change in the Upgrade guide?

Reply:
Added